test/parse/test_processing_instruction.rb

```diff
@@ -62,6 +43,28 @@ def test_xml_declaration_not_at_document_start
       end
     end
 
+    def test_garbage_text
+      doc = parse(<<~XML)
+      <?x y
+      <!--?><?x -->?>
+      <r/>
+      XML
+      assert_equal(["x", "y\n<!--"], [ doc.children[0].target, doc.children[0].content])
+      assert_equal(["x", "-->"], [ doc.children[1].target, doc.children[1].content])
```

Comment:
Could you unify these assertions to one assert for easy to debug?

```suggestion
      assert_equal([["x", "y\n<!--"],
                    ["x", "-->"]],
                   [[doc.children[0].target, doc.children[0].content],
                    [doc.children[1].target, doc.children[1].content]])
```

See also: https://www.clear-code.com/blog/2011/2/28.html

Reply:
OK.
I see.
It looks good.